### PR TITLE
Saratoga nyc housing prices

### PR DESCRIPTION
**Pull request by hongtonglin**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -12,7 +12,7 @@ Authors@R: c(
     person("Marium", "Tapal", email = "mariumtapal@gmail.com", role = "ctb", comment = c(ORCID = "0000-0001-5093-6462"))
     )
 Maintainer: Albert Y. Kim <albert.ys.kim@gmail.com>
-Description: Datasets and wrapper functions for tidyverse-friendly introductory linear regression, used in "Statistical Inference via Data Science: A ModernDive into R and the Tidyverse" available at <https://moderndive.com/>.
+Description: 🍒 Datasets and wrapper functions for tidyverse-friendly introductory linear regression, used in "Statistical Inference via Data Science: A ModernDive into R and the Tidyverse" available at <https://moderndive.com/>.
```
Comment (beanumber) [inline DESCRIPTION]: Did you mean to add a cherry emoji here? I don't think this will fly!

File: R/st_nyc_housing_prices.R [hunk 2]
```diff
@@ -0,0 +1,2 @@
+st_nyc_housing_prices <- read.csv("/Users/cynthialin/Downloads/saratoga_ny_home_prices (1).csv")
```
Comment (beanumber) [inline R/st_nyc_housing_prices.R]: Saratoga is not part of New York City. I'd suggest `saratoga_houses` as a name.

File: R/st_nyc_housing_prices.R [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+usethis::use_data(st_nyc_housing_prices, overwrite=TRUE)
```
Comment (beanumber) [inline R/st_nyc_housing_prices.R]: Please put spaces around `=`

File: R/st_nyc_housing_prices.R (hunk 2 above)
Comment (beanumber) [reply R/st_nyc_housing_prices.R]: Please pipe the URL into `read_csv()`

**Review by beanumber — CHANGES_REQUESTED** [review]

**Comment by beanumber** [comment]
@alejanmg @egordonhalpern are you seeing these messages?

File: DESCRIPTION [hunk 4]
```diff
@@ -12,10 +12,13 @@ Authors@R: c(
...
     person("Nikki", "Schuldt", email = "nikkischuldt@gmail.com", role = "ctb"),
-    person("Tess", "Goldmann", email = "tessgoldmann@aol.com", role = "ctb")
+    person("Tess", "Goldmann", email = "tessgoldmann@aol.com", role = "ctb"),
+    person("Hongtong", "Lin", email = "cccynthialht@gmail.com", role = "ctb"),
+    person("Alejandra", "Munoz", email = "amunozgarcia@smith.edu", role = "ctb"),
+    person("Elena", "Gordon-Halpern", role = "ctb")
     )
 Maintainer: Albert Y. Kim <albert.ys.kim@gmail.com>
-Description: Datasets and wrapper functions for tidyverse-friendly introductory linear regression, used in "Statistical Inference via Data Science: A ModernDive into R and the Tidyverse" available at <https://moderndive.com/>.
+Description: contributers: Hongtong, Alejandra, Elina
```
Comment (beanumber) [inline DESCRIPTION]: No. Please restore the previous Description of the package.

File: R/st_nyc_housing_prices.R [hunk 5]
```diff
@@ -0,0 +1,3 @@
+saratoga_houses <- read.csv("https://wjhopper.github.io/SDS-201/data/saratoga_ny_home_prices.csv")
```
Comment (beanumber) [inline R/st_nyc_housing_prices.R:1]: You want to put this code at the bottom of the `data-raw/process_data_sets.R` file. 

Please also put the URL on the first line and pipe the result into `read_csv()`.

**Comment by beanumber** [comment]
We still need to add to the NEWS.md, and write the documentation for the data set.

**Review by beanumber — CHANGES_REQUESTED** [review]

**Comment by hongtonglin** [comment]
i renamed my branch to sartaoga-houses. Do i need to make a new pull request??

**Pull request closed without merging**